addons/web/static/src/less/views.less

```diff
@@ -87,3 +87,71 @@
 .fa.o_toggle_button_success {
     color: @brand-success;
 }
+
+// Light labels (not in bold) for settings forms
```

Comment:
Shouldn't those rules be in form_view.less ? Those rules apply to form views only, and this file is cross-views. Maybe @ged-odoo could confirm.